### DISCUSSION TITLE
Off topic but..

### DISCUSSION THREAD

**Discussion by MisakeTheRat** [Prodigy-Hacking/ProdigyMathGameHacking#1239] [(A) Anything]
I've seen some people in game actually having convos like saying "birch wand" and stuff idfk what it means can someone help? please

**Comment by afkvido** [comment]
It's probbably random asf becuase Prodigy chat is so limited and we are truly oppresed. Maybe "Birch Wand" = "Bitch wand" and "Fungoid" = "Fuckhead". 

idk.

**Comment by afkvido** [comment]
Chances are people using the prodigy chat are just high, they all mustv'e been drinking vodka and smoking the bad shit

**Reply by MisakeTheRat** [reply]
Lmao thanks

**Reply by afkvido** [reply]
np!
